src/main/java/com/cinema/service/impl/MovieServiceImpl.java

```diff
@@ -1,17 +1,19 @@
 package com.cinema.service.impl;
 
 import com.cinema.dao.MovieDao;
-import com.cinema.lib.Inject;
-import com.cinema.lib.Service;
 import com.cinema.model.Movie;
 import com.cinema.service.MovieService;
 import java.util.List;
+import org.springframework.stereotype.Service;
 
 @Service
 public class MovieServiceImpl implements MovieService {
-    @Inject
     private MovieDao movieDao;
```

Comment:
let's use `final` everywhere or don't use it everywhere

Reply:
added everywhere